parcels/field.py

```diff
@@ -142,11 +143,13 @@ class Field(object):
     :param units: type of units of the field (meters or degrees)
     :param interp_method: Method for interpolation
     :param allow_time_extrapolation: boolean whether to allow for extrapolation
+    :param time_periodic: boolean whether to loop periodically over the time component of the dataset
+           This flags put automatically allow_time_extrapolation to False
```

Comment:
For clarity, call this 'Field', rather than 'dataset'? And also replace second line of comment with "This flag overrides the allow_time_interpolation and sets it to False".